ecommerce-app/src/styles/cart.module.scss

```diff
@@ -0,0 +1,45 @@
+/* cart.module.scss */
+.cartcontainer{
+  margin:10% 5%;
+}
+.cart-item {
+  display: flex;
+  margin-bottom: 20px;
+  border-bottom: 1px solid #ccc;
+  padding-bottom: 10px;
+
+  .product-details {
+    flex-grow: 1;
+    display: flex;
+    flex-direction: column;
+
+    p {
+      margin: 0;
+    }
+  }
+
+  .quantity-dropdown {
+    margin-right: 20px;
+
+    select {
+      margin-top: 5px;
+      padding: 5px;
+    }
+  }
+
+  .subtotal {
+    p {
+      margin: 0;
+      font-weight: bold;
+    }
+  }
+}
+
+.total {
+  text-align: right;
+  margin-top: 20px;
```

Comment:
consistency is important while defining margins and padding. We should rely on pre-defined variables so that spacings are consistent across the entire application.